### DISCUSSION TITLE
2021-08-27 Dev Platform

### DISCUSSION THREAD

**Discussion by jberrytech** [near/community#92] [Archived]
Updates related to Dev Platform team: https://github.com/near/dev-platform
Project board: https://github.com/orgs/near/projects/10

### Sandbox testing 
- Updated version of `runner-js`
- First version/prototype of `runner-as`
- Prototype for installing sandbox at test runtime 

### Hackathon Prep/Support
- Rust/NEAR guide version 2 published 
- 10+ cleanup tasks completed in prep of Hackathon 
  - Examples fixes
  - Documentation and link fixes 
- Integrations to CI to automate examples testing 

### Language Support 
- Grain NEAR support release imminent 

### Rust SDK
- 4.0 release coming along nicely, tracking issue here: https://github.com/near/near-sdk-rs/issues/494

### Hackathon Support 
- One DevPlatform member is monitoring hackathon and supporting DevRel each day 
- DevPlatform members have been participating in hackathon-focused office hours. @willemneal had a great support session

## runner-js
- Moving to convert existing sim tests to sandbox 
- Collaborating with DevRel on evaluation, testing and community messaging 

## runner-rs
- First internal demo of the Rust version of sandbox framework being executed this week 
- Collaborating with DevRel on evaluation, testing and community messaging 

## OWS / DevDashboard 
- Project plan and product mission being released this week.

**Comment by maxzaver** [comment]
Which version of runner-js is it referring to? The most recent one was 14 days ago https://github.com/near/runner-js/releases/tag/v0.5.2

**Reply by maxzaver** [reply]
Could you also add a link to runner-as repo?

**Reply by maxzaver** [reply]
Also, is this book v2 https://sdk-g4yv.onrender.com/ ?

**Reply by mikedotexe** [reply]
> Also, is this book v2 https://sdk-g4yv.onrender.com/ ?

I think Jim is referring to shepherding some docs tasks that DevRel leaned into for the hackathon, but as a quick note, the SDK Docs have a proper domain now at: https://www.near-sdk.io
